src/sql/pg/create.sql

```diff
@@ -0,0 +1,69 @@
+create table if not exists "accounts" (
+"id" serial not null primary key,
+"name" varchar(255),
+"balance" float check (balance >= minimum_allowed_balance),
+"connector" varchar(1024),
+"password_hash" varchar(1024),
+"public_key" text,
+"is_admin" boolean,
+"is_disabled" boolean,
+"fingerprint" varchar(255),
+"minimum_allowed_balance" float default 0);
+
+create unique index accounts_name_unique on "accounts" ("name");
+create index fingerprint on "accounts" ("fingerprint");
+
+
+create table if not exists "transfers" (
+"id" char(36) not null primary key,
```

Comment:
yes setting a column to PK will have a built in system generated index and unique constraint